src/allmydata/test/test_util.py

```diff
@@ -777,6 +788,12 @@ def test_get_pathinfo_symlink(self):
         self.failUnlessFalse(symlinkinfo.isfile)
         self.failUnlessFalse(symlinkinfo.isdir)
 
+    def test_encrypted_tempfile(self):
+        from allmydata.util.fileutil import EncryptedTemporaryFile
+        f = EncryptedTemporaryFile()
+        f.write("foobar")
+        f.close()
```

Comment:
maybe `with ...` or something... if it's a context manager.

Reply:
it is not, it seems :(